api/companies.py

```diff
@@ -129,14 +131,34 @@ def createCompany(json=None):
         wallet.money -= new_company_fee
         db_sess.merge(wallet)
         db_sess.commit()
-        return send_response(
+        send_response(
             event_name,
             {
                 'message': 'Success',
                 'errors': []
             }
         )
 
+        return send_response(
+            'showNotifications',
+            {
+                'message': 'Success',
+                'notifications': [
+                    {
+                        'logoSource': icons['new_company'],
+                        'author': news.author.split(' | ')[0],
+                        'company': None if len(news.author.split(' | ')) == 1
+                        else news.author.split(' | ')[1],
+                        'date': news.date.strftime('%d %B'),
+                        'time': news.date.strftime('%H:%M'),
+                        'redirectLink': f'{url("market.news")}#{news.id}'
+                    }
+                ],
+                'errors': []
+            },
+            broadcast=True, include_self=False
+        )
+
```

Comment:
Can't send result message success after notification, because it redirects to new company's post